### PR TITLE
Added description for the GLSL data types

### PR DESCRIPTION
**Pull request by ghost**
This is for the issue #19634

I removed this part 

**<\<document the basic types, vectors, etc.; see [Data Type (GLSL)](<https://www.khronos.org/opengl/wiki/Data_Type_(GLSL)>) on the Khronos WebGL wiki>>
>>>>>>> main**

and it is causing a merge conflict. 

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by wbamberg** [comment]
I've resolved the conflict: but please note that the PR that conflicts was merged 8 days ago: https://github.com/mdn/content/pull/19586/files , or almost a week _before_  you filed this PR. So please make sure you're working off an up-to-date version of mdn/content when you start a PR, otherwise this will happen a lot.

File: files/en-us/web/api/webgl_api/data/index.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ Shader programs have access to three kinds of data storage, each of which has a
 
 ## GLSL data types
 
-<\<document the basic types, vectors, etc.; see [Data Type (GLSL)](<https://www.khronos.org/opengl/wiki/Data_Type_(GLSL)>) on the Khronos WebGL wiki>>
+Basic types as the integral parts of the [GLSL data types](https://www.khronos.org/opengl/wiki/Data_Type_(GLSL)) are scalars, vectors, swizzling, matrices, opaque types, and Implicit conversion. 
+These types can be categorized into other variants, such as arrays. GLSL has other array types known as Opaque arrays, and Arrays of arrays. Other main GLSL data types include Structs, Constructors and initializers, and Interface blocks.
```
Comment (wbamberg) [inline files/en-us/web/api/webgl_api/data/index.md:27]:
```suggestion
See [Data Types](https://www.khronos.org/opengl/wiki/Data_Type_(GLSL)) in the GLSL documentation.
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for you PR!

I think it's better just to have a link here.

**Comment by najmiehsa** [comment]
> I've resolved the conflict: but please note that the PR that conflicts was merged 8 days ago: https://github.com/mdn/content/pull/19586/files , or almost a week _before_ you filed this PR. So please make sure you're working off an up-to-date version of mdn/content when you start a PR, otherwise this will happen a lot.

OK sure.

**Comment by najmiehsa** [comment]
I think I need to edit this maybe. Please let me know ^^

**Review by wbamberg — APPROVED** [review]
👍 thank you @najmiehsa !

**Comment by najmiehsa** [comment]
Thank you.

**Pull request merged**